data-processing/common/upload_entities.py

```diff
@@ -309,3 +313,53 @@ def resolve_model(entity_identifier: EntityIdentifier) -> Optional[Entity]:
                     )
 
     return models
+
+
+def write_to_file(entity_infos: List[EntityUploadInfo], output_file_name: str) -> None:
+    # an attempt to make life easier if we change how we want to format this
+    FORMAT_VERSION = "v0"
```

Comment:
Love the versioning! It will be great to be able to distinguish between data created by different versions of the pipeline.

Reply:
Just to make sure we're on the same page, I see this as more about the versioning of the format of what gets written. So even if we updated say the citations extraction code, I wouldn't expect _this_ version to necessarily change (though s2airs would track that through a different path). And the other side of that is that this version could change even if the extraction code doesn't, say if we decide we want our json output to have a different nesting structure, or change some of the key names.

Reply:
(I ended up changing the key name from 'version' to 'format_version' to try to make this clearer.)

Reply:
This sounds good!